### PR TITLE
LibWeb/LibGfx: Implement CanvasRenderingContext2D: isPointInPath() 

### PR DESCRIPTION
**Pull request by mierenhoop**

### CODE REVIEW

**Comment by awesomekling** [comment]
Welcome to the Ladybird project! Great idea for a first PR :)

Please squash the clang-format changes into the relevant preceding commit(s) instead of adding them as a separate commit.

**Comment by mierenhoop** [comment]
The commits have been remade and force-pushed, hopefully it's not confusing.

File: Userland/Libraries/LibWeb/HTML/Canvas/CanvasDrawPath.idl [hunk 1]
```diff
@@ -18,8 +18,8 @@ interface mixin CanvasDrawPath {
     // FIXME: `DOMString` should be `CanvasFillRule`
     undefined clip(Path2D path, optional DOMString fillRule = "nonzero");
 
-    [FIXME] boolean isPointInPath(unrestricted double x, unrestricted double y, optional CanvasFillRule fillRule = "nonzero");
-    [FIXME] boolean isPointInPath(Path2D path, unrestricted double x, unrestricted double y, optional CanvasFillRule fillRule = "nonzero");
```
Comment (AtkinsSJ) [inline Userland/Libraries/LibWeb/HTML/Canvas/CanvasDrawPath.idl:22]: Please add the same
```
// FIXME: `DOMString` should be `CanvasFillRule`
```
above each of these that you see a few lines above.

**Review by AtkinsSJ — CHANGES_REQUESTED** [review]
Just one nit-pick, otherwise this looks good. (Again, please squash the change into the existing commit.)

**Comment by mierenhoop** [comment]
Surprisingly the CI fails... After adding the comments.

**Comment by AtkinsSJ** [comment]
> Surprisingly the CI fails... After adding the comments.

We've been having the macOS CI time out for a while, nobody's figured out what the cause is. But it's not related to your changes. :^)

**Pull request merged**